Fixed top-level readme w/ more detailed guideline (#105)
informalsystems/ibc-rs: README.md

> 4. Start the light clients and a dummy relayer thread:

Author: I'm not really sure what you mean by "event information". Do you mean the first two lines in the following log output?

```
Jun 15 17:55:00.601  INFO relayer::event_handler: Chain chain_B pushed {"NewBlock":{"height":"14057"}}
Jun 15 17:55:01.082  INFO relayer::event_handler: Chain chain_A pushed {"NewBlock":{"height":"14058"}}
....
Jun 15 19:46:21.524  INFO relayer_cli::commands::start: client updated to trusted state: 690C0969D66E6A4C378608A0AC7AE098AA4CDA20B87F363C276EACFF94FD2FE7 at height 20167 chain.id=chain_A
Jun 15 19:46:21.528  INFO relayer_cli::commands::start: client updated to trusted state: 26F4894926EE2A769737634239340CED649A64EDF9DF8926C9FAB892577F26C3 at height 20166 chain.id=chain_B
```

> 4. Start the light clients and a dummy relayer thread:

Reviewer: If you run with `-v` you will see the events received by the relayer thread from the monitors.

    ```bash
    ibc-rs $ cargo run --bin relayer -- -c ./relayer/relay/tests/config/fixtures/relayer_conf_example.toml start --reset
    ```

    The `--reset` flag only needs to be passed once, for initializing the trusted headers based on the hash & height stored from steps 1-3 above.

> Beside the basic relayer `start` command, other subcommands are also supported at the moment:

Reviewer: ```suggestion
Beside the basic relayer `start` command, the following are also available:
```

- `query`can be used to initiate various queries against one of the chains, for example: `cargo run --bin relayer -- -v -c ./relayer/relay/tests/config/fixtures/relayer_conf_example.toml query connection end A Z` will look up the connection with identifier `Z` on chain `A`.

The `relayer/cli/src/commands.rs` file contains further description of the CLI subcommands.

**Note:** Add a `-v` flag to the commands above to see detailed log output, eg. `cargo run --bin relayer -- -v -c ./relayer/relay/tests/config/fixtures/relayer_conf_example.toml run`

### Development environment

To set up a local development environment, clone the relayer implementation from [iqlusioninc/relayer](https://github.com/iqlusioninc/relayer/).
We are interested in two commands we can run from this repo:

- `bash scripts/two-chainz "local" "skip"`. Running this script will instantiate two chains, listening on ports `26557` and `26657`, respectively.

- `bash dev-env`. Running this script script from your local source will instantiate two chains, on ports `26557` and `26657`, and also a relayer that sets up some connections and channels.

> Note that these script rely on the [cosmos/gaia](https://github.com/cosmos/gaia) implementation, which provides the basic tools for setting up a IBC-enabled Tendermint chain (called a "Cosmos Hub" in the IBC terminology).

Reviewer: ```suggestion
Note that these script rely on the [cosmos/gaia](https://github.com/cosmos/gaia) implementation, which is a Cosmos-SDK application for the cosmos hub. 
```
